scripts/esmDependencyCheck.js

```diff
@@ -0,0 +1,226 @@
+// eslint-disable no-console
+const { exec } = require('child_process');
+const fs = require('fs');
+const fetch = require('node-fetch');
+const dotenv = require('dotenv').config();
+const { dependencies, devDependencies } = require('../package.json');
```

Comment:
Can we use `import` instead of `require`?

Reply:
@eagerterrier says 'node-fetch is central dependency that we have tied to commonjs version 2.6.12 whereas v3 is esm 3.3.2. So don’t think we can use module style import until node-fetch is on v3'

Reply:
Why do we need to use node-fetch? Aren't these scripts being ran in a Node 18 environment which has a Fetch API built in?

Reply:
I have a PR that attempts to remove node-fetch in general: https://github.com/bbc/simorgh/pull/11122. Certainly seems doable given we're on Node 18. The Express app would just need a bit of work to work with Undici.

Although as you say, this is ran purely in the Node environment outside of the browser, so we could likely remove it from here.